src/services/uniswapV3/LiquidityService.ts

```diff
@@ -0,0 +1,415 @@
+      return await this.fetchLiquidityTicks(poolAddress, concatTicks, limit, limit + skip);
+    }
+
+    return ticks.concat(response.data.data.ticks);
+  }
+
+  async getPoolTickSum(poolAddress: string, token0: Token, token1: Token, fee: FeeAmount, chainId: ChainsIds) {
+    const provider = this.blockchainProviderRepository.get(chainId);
+    const poolContract = new ethers.Contract(poolAddress, IUniswapV3PoolABI.abi, provider);
+    const [slot0, liquidity, graphTicks] = await Promise.all([
+      poolContract.slot0(),
+      poolContract.liquidity(),
+      this.getFullTickData(poolAddress),
+    ]);
+
+    const sdkTicks = graphTicks.map((graphTick: GraphTick) => {
+      return new Tick({
+        index: +graphTick.tickIdx,
+        liquidityGross: graphTick.liquidityGross,
+        liquidityNet: graphTick.liquidityNet,
+      });
+    });
+
+    const tickSpacing = TICK_SPACINGS[fee];
+
+    const tickIdxToTickDictionary: Record<string, GraphTick> = Object.fromEntries(
+      graphTicks.map((graphTick) => [graphTick.tickIdx, graphTick]),
+    );
+
+    console.log('liquidity: ', graphTicks[0]);
+
+    const fullPool = new Pool(token0, token1, fee, slot0.sqrtPriceX96, liquidity, slot0.tick, sdkTicks);
+
+    const activeTickIdx = Math.floor(fullPool.tickCurrent / tickSpacing) * tickSpacing;
+
+    const currentTickInitialized = tickIdxToTickDictionary[activeTickIdx];
+
+    console.log('currentTickInitialized: ', currentTickInitialized);
+    const activeTickProcessed: TickProcessed = {
+      tickIdx: activeTickIdx,
+      liquidityActive: liquidity,
+      liquidityNet: JSBI.BigInt(0),
+      price0: parseFloat(tickToPrice(token0, token1, activeTickIdx).toSignificant(18)),
+      price1: parseFloat(tickToPrice(token1, token0, activeTickIdx).toSignificant(18)),
+      isCurrent: true,
+    };
+
+    const activeTick = tickIdxToTickDictionary[activeTickIdx];
+    if (activeTick) {
+      activeTickProcessed.liquidityNet = JSBI.BigInt(activeTick.liquidityNet);
+    }
+
+    const numSurroundingTicks = 100;
+
+    const createBarChartTicks = await this.createBarChartTicks(
+      activeTickIdx,
+      fullPool.liquidity,
+      tickSpacing,
+      fullPool.token0,
+      fullPool.token1,
+      numSurroundingTicks,
+      fullPool.fee,
+      graphTicks,
+    );
+
+    const poolSum = createBarChartTicks.reduce(
+      (acc, curr) => {
+        acc = {
+          liquidityActive: JSBI.toNumber(JSBI.add(JSBI.BigInt(acc.liquidityActive), JSBI.BigInt(curr.liquidityActive))),
+          liquidityLockedToken0: acc.liquidityLockedToken0 + curr.liquidityLockedToken0,
+          liquidityLockedToken1: acc.liquidityLockedToken1 + curr.liquidityLockedToken1,
+        };
+
+        return acc;
+      },
+      {liquidityActive: 0, liquidityLockedToken0: 0, liquidityLockedToken1: 0},
+    );
+
+    console.log('poolSum: ', poolSum);
+
+    return poolSum;
+  }
+
+  async createBarChartTicks(
```

Comment:
this is entry point from SDK example right? let's create separate class for code we copied from uniswap